aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSampler.java

```diff
@@ -72,9 +76,12 @@ public static AwsXrayRemoteSamplerBuilder newBuilder(Resource resource) {
 
     sampler = initialSampler;
 
-    pollFuture =
-        executor.scheduleAtFixedRate(
-            this::getAndUpdateSampler, 0, pollingIntervalNanos, TimeUnit.NANOSECONDS);
+    this.pollingIntervalNanos = pollingIntervalNanos;
+    // Add ~1% of jitter. Truncating to int is safe for any practical polling interval.
```

Comment:
Currently 5 sec of jitter is 1.6666% while 10ms for the target poller is 1%. 3s seems fine to me, especially since we compute the jitter with nanosecond precision instead of seconds precision like the xray sdk